cmd/kogito/command/flag/podresource_flag.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package flag
+
+import (
+	"fmt"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/util"
+	"github.com/spf13/cobra"
+)
+
+// PodResourceFlags is common properties used to configure CPU resources
+type PodResourceFlags struct {
+	Limits   []string
+	Requests []string
+}
+
+// AddPodResourceFlags adds the common resource flags to the given command
+func AddPodResourceFlags(command *cobra.Command, flags *PodResourceFlags, prefix string) {
+	limitName := "limits"
+	requestName := "requests"
+	if prefix != "" {
```

Comment:
```suggestion
	if len(prefix) > 0 {
```

We are using this method to check for empty strings.

Reply:
done